src/main/java/at/hannibal2/skyhanni/config/features/fishing/TotemOfCorruptionConfig.java

```diff
@@ -0,0 +1,50 @@
+package at.hannibal2.skyhanni.config.features.fishing;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.config.core.config.Position;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorColour;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorSlider;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class TotemOfCorruptionConfig {
+
+    @Expose
+    @ConfigOption(name = "Show Overlay", desc = "Show the Totem of Corruption overlay." +
```

Comment:
This feature is the main toggle. either rename this to enabled, or notice in all other features below that this one needs to be enabled first.